mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -194,10 +215,13 @@ pub async fn retry_subscriptions(
 
     while let Some(subs) = retries.next().await {
         if !subs.is_empty() {
-            info!("try to re-subscribe to {} topics", subs.len());
+            warn!("try to re-subscribe to {} topics", subs.len());
             for sub in subs {
                 if topic_mappers_updates.contains_key(&sub.topic_filter) {
-                    debug!("re-subscribe to {} qos {:?}", sub.topic_filter, sub.qos);
+                    warn!(
+                        "subscription was rejected by server, re-subscribe to {} qos {:?}",
```

Comment:
Do we know here that the sub was rejected by the server? Should we log it when it happens instead of here?
Nit - suggested text: Re-subscribing to topic {} with qos {}

Reply:
it is because it was rejected because the way is retried now, but removed that part because it makes more sense to log when the reject is received